modules/ROOT/pages/update-workspace.adoc

```diff
@@ -3,9 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Studio 7.2 and later simplifies the configuration of the `mule-artifact.json` file. This file is used when packaging your Mule application to identify the class loader configuration and settings of your Mule application, and the minimum required Mule runtime version to run your application. +
+Anypoint Studio 7.2 and later simplifies the configuration of the `mule-artifact.json` file. This file is used when packaging your Mule application to identify the class loader configuration and settings of your Mule application, and the minimum required Mule runtime engine version to run your application.
 
-Anypoint Studio includes a new version of the Mule Maven Plugin that can automatically infer some of this required metadata. This make it easier for you to keep track of all the declarations you need to make when exporting resources. +
+Anypoint Studio includes a new version of the Mule Maven Plugin that can automatically infer some of this required metadata. This make it easier for you to keep track of all the declarations you need to make when exporting resources.
```

Comment:
Is the plugin a Mule plugin or an Apache Maven plugin?
```suggestion
Anypoint Studio includes a new version of the Mule plugin for Apache Maven that can automatically infer some of this required metadata. This makes it easier for you to keep track of all the declarations you need to make when exporting resources.
```
OR
```suggestion
Anypoint Studio includes a new version of the Mule implementation of the Apache Maven plugin that can automatically infer some of this required metadata. This makes it easier for you to keep track of all the declarations you need to make when exporting resources.
```

Reply:
@kbostrom "Mule Maven Plugin" is the name of the plugin. Probably not an official name product, but that's how it's been known for years: https://docs.mulesoft.com/mule-runtime/4.2/mmp-concept